.github/problem-matchers/modpost.json

```diff
@@ -0,0 +1,16 @@
+{
+    "problemMatcher": [
+        {
+            "__comment_owner": "match modpost warnings/errors",
+            "owner": "modpost",
+            "pattern": [
+                {
+                    "__comment_regexp1": "ERROR: modpost: devm_drm_of_get_bridge [drivers/gpu/drm/vc4/vc4.ko] undefined!",
```

Comment:
Hah. Someone should send a patch to reverse the word order here. :) It really should be: `modpost: ERROR: ...` etc to match basically all the other tools. :)